### ISSUE TITLE
Cannot bind cmd-tab on OSX

### ISSUE THREAD

**Issue by zumbalogy** [atom/atom-keymap#70]
'.platform-darwin':
  'cmd-tab': 'tree-view:toggle'

cmd-tab (under any selector), does not seem to set the command. I actually want to just unset it, because atom is getting the command and inserting an  unknown character box type thing. it does this for some other commands as well (ctrl-alt-left, for example), but those commands are not ones I use. Any  workaround or hack is welcome, I just need it to stop doing this if I am going to be able to use atom.

**Comment by 50Wliu** [comment]
What keyboard layout are you using?
Also I believe you can just do `'cmd-tab': 'unset!'`.

**Comment by zumbalogy** [comment]
I am using qwerty, US english, and I have confirmed that both cmd and tab are working when I bind them, but together it does not work, and it inserts a bad character. My system still gets the keypress too though, so it does tab me to a different window (like I want it to. But i don't want random invalid characters being typed)

**Comment by nathansobo** [comment]
Doesn't `cmd-tab` activate your application switcher, or did you change that default binding?

**Comment by zumbalogy** [comment]
I have cmd-tab cycle between active windows on my machine (this is also all on an external keyboard (of brand 'wasd')), but my machine does still get the command and it does still cycle my to a different window like I want it to. But before it does this, a [bad] character is inserted.

for those googleing, this is a hack, but i fixed it by putting this in my init.coffee
atom.workspace.observeTextEditors (editor) ->
  editor.onWillInsertText (edit) ->
    edit.cancel() if edit.text == ''


(the square will have to be the same square as is inserted, so others may have to copy paste their specific one)

**Comment by nathansobo** [comment]
@zumbalogy Ah, so to be clear you've changed the default for `cmd-tab` to switch between active windows instead of applications?

**Comment by zumbalogy** [comment]
Yes, and that behavior still works, its just that it also inserts that little unknown character thing. ctrl-alt-left also inserts this little funny character, but I don't mind that. My workaround works for now, but it would be nice if I could just unset it or set it to a null command or something.
